report.Rmd

```diff
@@ -53,19 +53,25 @@ long_rt <- utla_rt_cases %>%
   filter(!is.na(rt_mean))
 ```
 
+1. Center for the Mathematical Modelling of Infectious Diseases, London School of Hygiene & Tropical Medicine, London WC1E 7HT, United Kingdom
+
 For correspondence: sebastian.funk@lshtm.ac.uk
 
-## Abstract
+# Abstract
 
-**Background:** Local estimates of the time-varying effective reproduction number (Rt) of Covid-19 in England become increasingly heterogeneous in England during April and May. This variation may have been attributed to the spread of novel SARS-CoV-2 variant "Delta". In this report we aimed to investigate the changes in the proportion of positive cases that were S-gene positive, an indicator of the novel variant in front of the background of the previously predominant variant "Alpha", and to correlate them with changes in the estimated reproduction number at the level of upper tier local authorities (UTLA).
+**Background:** Local estimates of the time-varying effective reproduction number (Rt) of Covid-19 in England became increasingly heterogeneous in England during April and May 2021. This variation may have been attributed to the spread of novel SARS-CoV-2 variant "Delta". In this study we aimed to investigate the changes in the proportion of positive cases that were S-gene positive, an indicator of the novel variant in front of the background of the previously predominant variant "Alpha", and to correlate them with changes in the estimated reproduction number at the level of upper tier local authorities (UTLA).
 
-**Method:** We explored the relationship between the proportion of samples that were S-gene positive and the effective reproduction number of test-positive cases over time. Effective reproduction numbers were estimated using the `EpiNow2` R package independently for each local authority using two different estimates of the generation time. The `brms` R package was used to fit a range of regression models to estimate a multiplicative relationship between S-gene positivity and weekly mean R estimate. Results are shown with 90% credible intervals.
+**Method:** We explored the relationship between the proportion of samples that were S-gene positive and the effective reproduction number of test-positive cases over time. Effective reproduction numbers were estimated using the `EpiNow2` Rt package independently for each local authority using two different estimates of the generation time. The `brms` Rt package was used to fit a range of regression models to estimate a multiplicative relationship between S-gene positivity and weekly mean Rt estimate. Results are shown with 90% credible intervals.
```

Comment:
Find+replace fail ;-)
Can you do find "Rt package" -> replace with "R package"?

Reply:
And afterwards maybe "Rt" -> "$R_t$"?